src/main/java/org/humancellatlas/ingest/process/Process.java

```diff
@@ -24,7 +24,7 @@
  * Created by rolando on 16/02/2018.
  */
 @Getter
-@EqualsAndHashCode(callSuper = true)
+@EqualsAndHashCode(callSuper = true, exclude = {"project", "projects", "protocols", "inputBundleManifests", "chainedProcesses"})
```

Comment:
I don't think the chainedProcesses are being used at all, could just remove that field in the future.